sample_factory/algorithms/appo/learner.py

```diff
@@ -446,6 +445,26 @@ def _prepare_train_buffer(self, rollouts, macro_batch_size, timing):
                 if isinstance(x[0], (dict, OrderedDict)):
                     buffer[key] = list_of_dicts_to_dict_of_lists(x)
 
+        # Add max entropy to the rewards
+        with torch.no_grad():
+            ent_coeff = self.cfg.ppo_max_entropy_coeff
+            # flatten the action dist parameters in the batch
+            logits = np.array(buffer.action_logits) # [E, T, A]
```

Comment:
@alex-petrenko: changes to fix tensor/array reshaping bugs, in two similar ways. Please see which one works better for the coding style in SF.

Note that this is inefficient since we have to reshape it again to match with the shape of the rewards (`np.stack(buffer.rewards).squeeze().shape`)